### DISCUSSION TITLE
Audio problems after last update

### DISCUSSION THREAD

**Discussion by Kostik1982** [aclap-dev/video-downloadhelper#740] [Help and Q&A]
### Your question

Hello. I am facing an issue with downloads after the last app update. Some of the videos are downloaded without sound and when i try to download the audio file separately i get an error message. Is there a workaround on this?

### Addon & CoApp version and details

Hello. I am facing an issue with downloads after the last app update. Some of the videos are downloaded without sound and when i try to download the audio file separately i get an error message. Is there a workaround on this?

**Comment by GrampaWildWilly** [comment]
You really should provide a proper problem report by following the form you ignored when you posted this.  It is important to know what releases of the various pieces of software you are running.  It's also important to supply the URL of a specific web page on which this problem is occurring.  If you say it's "everywhere," at least post a representative example.  We need a URL of a web page that contains a video you are having trouble downloading.  Don't just name a site.  Give a specific URL.

**Reply by TonyginSLC** [reply]
@GrampaWildWilly  You may want to try being a bit less condescending when you post. It is quite rude.
